x/crosschain/client/cli/tx_vote_inbound.go

```diff
@@ -87,3 +93,16 @@ func CmdVoteInbound() *cobra.Command {
 
 	return cmd
 }
+
+func parseProtocolContractVersion(version string) (types.ProtocolContractVersion, error) {
+	switch version {
+	case "V1":
+		return types.ProtocolContractVersion_V1, nil
+	case "V2":
+		return types.ProtocolContractVersion_V2, nil
```

Comment:
nit: Technically golang convention for constants is snake case without any _.

Reply:
This is code generated by proto when using `enum`